tests/vmi_multus_test.go

```diff
@@ -155,9 +157,8 @@ var _ = Describe("Multus", func() {
 		CIDR1 := "10.1.1.1/24"
 		CIDR2 := "10.1.1.2/24"
 		Context("VirtualMachineInstance with cni ptp plugin interface", func() {
-			testIP := cidrToIP(CIDR1)
```

Comment:
Looks like in you 2nd commit you reverted things from the 1st commit.

Reply:
Thanks. will fix this.